app/styles/_overview.less

```diff
@@ -97,59 +106,94 @@
   .standalone-service {
     width: 100%;
     @media (min-width: @screen-sm-min) {
-      width: 50%;
-
-      &:nth-child(odd) {
-          padding-right: 5px;
-      }
-
-      &:nth-child(even) {
-          padding-left: 5px;
-      }
-
+      .flex(@columns: 0 0 49%);
+    } 
+    @media (min-width: @screen-md-min) {
+      .flex(@columns: 0 0 49.375%);
```

Comment:
Since it uses flex-basis to set percentage widths for the 2 columns and the remainder is for the gutter in between, padding and negative margins are no longer needed for gutters and positioning. To maintain an approximate gutter width around 15px the flex-basis is adjusted slightly larger when > 992px.